test/ui-test/src/screens/SettingsScreen.py

```diff
@@ -88,4 +94,28 @@ def _find_account_index(self, account_name: str) -> int:
         for index in range(accounts.count):
             if(accounts.itemAtIndex(index).objectName == account_name):
                 return index
-        return -1
\ No newline at end of file
+        return -1
+
+    def select_default_account(self):
+        accounts = get_obj(WalletSettingsScreen.GENERATED_ACCOUNTS.value)
+        click_obj(accounts.itemAtIndex(0))
+        click_obj_by_name(WalletSettingsScreen.EDIT_ACCOUNT_BUTTON.value)
+
+    def edit_account(self, account_name: str, account_color: str):
+        type(WalletSettingsScreen.EDIT_ACCOUNT_NAME_INPUT.value, account_name)
+        colorList = get_obj(WalletSettingsScreen.EDIT_ACCOUNT_COLOR_REPEATER.value)
+        for index in range(colorList.count):
+            color = colorList.itemAt(index)
+            if(color.radioButtonColor == account_color):
+                click_obj(colorList.itemAt(index))
+
+        click_obj_by_name(WalletSettingsScreen.EDIT_ACCOUNT_SAVE_BUTTON.value)
+
+
+    def verify_editedAccount(self, new_name: str, new_color: str):
+        accountName = get_obj(WalletSettingsScreen.ACCOUNT_VIEW_ACCOUNT_NAME.value)
+        iconSettings = get_obj(WalletSettingsScreen.ACCOUNT_VIEW_ICON_SETTINGS.value)
+        verify_equal(accountName.text, new_name) and verify_equal(iconSettings.icon.color, new_color.lower())
```

Comment:
Could you move those to two different lines. That way, if only one of them fails, it's gonna be easier to tell which one is faulty.

Also, maybe we can improve `verify_equal` to have a better error message that prints the values of the objects,
or you can use my new function that I added (not merged yet): https://github.com/status-im/status-desktop/pull/6916/files#diff-2dd7446781e3cb54a56fc3cb79727ffbfb80c9c1471820a936b781082a1bee83R30

Reply:
I introduced the same function in my commit :D also all done, review when you can :)